src/components/Homepage/HomePage.test.js

```diff
@@ -0,0 +1,83 @@
+import { fireEvent, render, screen } from "@testing-library/react";
+import axios from "axios";
+import { MemoryRouter } from "react-router-dom";
+import HomePage from "./HomePage";
+import "@testing-library/jest-dom";
+
+jest.mock("axios");
+
+const MockHomePage = () => {
+  return (
+    <MemoryRouter>
+      <HomePage />
+    </MemoryRouter>
+  );
+};
+
+describe("Homepage", () => {
+  let response;
+  beforeEach(() => {
+    response = {
+      data: {
+        results: [
+          { title: "first", id: 1 },
+          { title: "second", id: 2 },
+          { title: "third", id: 3 },
+          { title: "fourth", id: 4 },
+          { title: "fifth", id: 5 },
+        ],
+      },
+    };
+  });
+
+  it("should render header when render homepage", async () => {
```

Comment:
You don't  need the `async` keyword in this test.

Reply:
fixed